lib/onfido/version.rb

```diff
@@ -1,3 +1,3 @@
 module Onfido
-  VERSION = '1.1.0'.freeze
+  VERSION = '1.2.0'.freeze
```

Comment:
I think we need to update the README too?

Reply:
Not anymore, switch it back to 1.1.0 :)